### DISCUSSION TITLE
How to disable buffers of nvim-tree or other windowed plugins

### DISCUSSION THREAD

**Discussion by tony-sol** [nvim-lualine/lualine.nvim#1112] [Q&A]
TL;DR - see only file buffers, even if cursor in nvim-tree or symbols-outline window

<img width="999" alt="nvim-tree" src="https://github.com/nvim-lualine/lualine.nvim/assets/33671815/f223f84f-dd1b-489c-9bc3-58b28751725e">
<img width="999" alt="buffer" src="https://github.com/nvim-lualine/lualine.nvim/assets/33671815/f5fd8790-d423-412c-aaaf-e26311e9507e">
<img width="999" alt="outline" src="https://github.com/nvim-lualine/lualine.nvim/assets/33671815/74c98af3-3103-4132-92b9-8a01f733c8b4">

---

I have a `buffers` component in `lualine_a` section of `tabline` configured like
```lua
lualine_a = {
	{
		'buffers',
		mode            = 4,
		use_mode_colors = true,
	},
},
```
When i have 3 windows opened, like 
```
nvim-tree | some file | symbols-outline
```
displayed buffers changed, depends on cursor position - when cursor is in main window with opened file, there are no extra buffers displayed, but when cursor moves to nvim-tree or symbols-outline - there is additional buffer appears.
Is there a way to disable this and do not add extra `Nvim_Tree_1`, `OUTLINE`, etc. buffers, wherever cursor is?
